apps/emqx_durable_storage/src/emqx_ds_builtin_db_sup.erl

```diff
@@ -150,3 +166,21 @@ egress_spec(DB, Shard) ->
         restart => permanent,
         type => worker
     }.
+
+cache_coordinator_spec(DB, Opts) ->
+    #{
+        id => {coordinator, DB},
+        start => {emqx_ds_cache_coordinator, start_link, [DB, Opts]},
+        shutdown => 5_000,
+        restart => permanent,
+        type => worker
+    }.
+
+cache_worker_sup_spec(DB) ->
```

Comment:
Is it possible to use `sup_spec` function here as well for the sake of consistency?